### DISCUSSION TITLE
System Requirements Specification (SRS) document

### DISCUSSION THREAD

**Discussion by TemirlanAidarov** [webdriverio/webdriverio#8053] [Q&A] [answered]
I have just found about this project.
I am Computer Science student who takes a course related to Open Software Project.
We have an assignment to write/improve System Requirements Specification (SRS) document for any project. Basically, the SRS is a document that describes the nature of a project, software or application. In simple words, SRS document is a manual of a project provided it is prepared before you kick-start a project/application.

Can I write/improve SRS document for this project?

**Comment by christian-bromann** [comment, marked as answer]
> Can I write/improve SRS document for this project?

Absolutely! I am honored that you picked WebdriverIO for this. How can I help you with it?
